### PR TITLE
Update gitea to build from source, OSX and Linux only

### PR DESCRIPTION
**Pull request by manics**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes https://github.com/conda-forge/gitea-feedstock/issues/3

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,34 +1,45 @@
...
   run:
     - git
 
 test:
   commands:
+    - gitea --version
     - gitea --help
 
 about:
   home: https://gitea.io
```
Comment (techknowlogick) [inline recipe/meta.yaml:38]: Could you update this URL to https://about.gitea.com ?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -38,4 +49,5 @@ about:
```
Comment (techknowlogick) [inline recipe/meta.yaml:49]: We've also updated the docs to be https://docs.gitea.com

File: recipe/meta.yaml (hunk 1 above)
Comment (manics) [reply recipe/meta.yaml]: Done

File: recipe/meta.yaml (hunk 2 above)
Comment (manics) [reply recipe/meta.yaml:51]: Done

**Comment by manics** [comment]
I couldn't get this to build on Windows (I presume upstream probably uses cross-compilation to create the Windows executable), so this is MacOS and Linux only.

@techknowlogick Would you like to be added as a co-maintainer of this package?

**Comment by techknowlogick** [comment]
@manics yup, that's right we do use cross compiling upstream for windows. I'm happy to be added as a co-maintainer, thanks!

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1 @@
+* @manics @nehaljwani
```
Comment (techknowlogick) [inline .github/CODEOWNERS:1]: Could you add me in here too?

**Review by techknowlogick — APPROVED** [review]
🎉

**Comment by manics** [comment]
@conda-forge/core please could you review this?

**Comment by techknowlogick** [comment]
Thanks for your work on this PR @manics, and also thanks to @ocefpaf for merging this <3

**Pull request merged**